### PR TITLE
[DEX] Initial Orders View Setup

### PR DESCRIPTION
**Pull request by schaeferCode**
## Screenshots
screen shot 2018-09-27 at 6 01 38 pm

### CODE REVIEW

File: src/components/dex/Orders.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+</template>
+
+<script>
+
+import OrdersTabs from './OrdersTabs';
+import MyOrders from './MyOrders';
+
+const MY_ORDERS = 'my-orders';
+const ORDER_BOOK = 'order-book';
+const TRADE_HISTORY = 'trade-history';
```
Comment (crishellco) [inline src/components/dex/Orders.vue:17]: I assume `ORDER_BOOK` and `TRADE_HISTORY` are going to be used eventually?

File: src/components/dex/OrdersTable.vue [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  <section id="dex--orders-table">
+    <div class="header">
+      <div v-for="status in orderStatus"
+        :class="[status, { active: selectedStatus === status }]"
+        @click="handleStatusChange(status)">
+        {{ status }} (3)
+      </div>
+    </div>
+    <aph-simple-table v-bind="{ data: ordersTableData, columns, hasHeader: false }">
+      <div slot="pairAndSide" slot-scope="{value}" class="pair-and-side-cell">
```
Comment (crishellco) [inline src/components/dex/OrdersTable.vue:11]: Nicely done with `slots` and `scopes`.

File: src/components/dex/OrdersTable.vue [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+            </div>
+            <div class="base-currency">
+              {{ value.base }}
+            </div>
+          </div>
+          <div class="price-cost">
+            <div class="price">
+              {{ value.price }}
+            </div>
+            <div class="cost">
```
Comment (crishellco) [inline src/components/dex/OrdersTable.vue:33]: You can probably one-line all of these.

File: src/components/dex/OrdersTable.vue [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+  .pair-and-side-cell {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    justify-content: flex-start;
+
+    .side {
+      margin-top: $space-sm;
+
+      &.Buy {
```
Comment (crishellco) [inline src/components/dex/OrdersTable.vue]: ew

File: src/components/dex/OrdersTable.vue [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+        color: $green;
+        
+        &:before {
+          content: "+";
+        }
+      }
+
+      &.Sell {
```
Comment (crishellco) [inline src/components/dex/OrdersTable.vue]: ew

**Review by crishellco — APPROVED** [review]

File: src/components/dex/Orders.vue (hunk 1 above)
Comment (schaeferCode) [reply src/components/dex/Orders.vue:17]: Yeah.

File: src/components/dex/OrdersTable.vue (hunk 2 above)
Comment (schaeferCode) [reply src/components/dex/OrdersTable.vue:11]: Pretty cool feature!

File: src/components/dex/OrdersTable.vue (hunk 4 above)
Comment (schaeferCode) [reply src/components/dex/OrdersTable.vue]: HAHA! Cause the title case?

**Pull request merged**